python/pyiceberg/io/fsspec.py

```diff
@@ -136,8 +136,6 @@ class FsspecFileIO(FileIO):
     """A FileIO implementation that uses fsspec"""
 
     def __init__(self, properties: Properties):
-        self._scheme_to_fs = {}
-        self._scheme_to_fs.update(SCHEME_TO_FS)
```

Comment:
This allows overriding the scheme-to-fs mapping for an individual FileIO instance. Do we want to remove that capability?

Reply:
How would this enable this? The `_scheme_to_fs` should not be accessed from outside of the class, and currently we just take an exact copy of it..